app/crud.py

```
@@ -0,0 +1,65 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+
+from .models import ToDo
+
+
+async def create_todo(db: AsyncSession, todo: ToDo):
+    try:
+        db.add(todo)
+        await db.commit()
+        await db.refresh(todo)
+        return todo
```

Ruff (0.8.2)

Consider moving this statement to an `else` block

(TRY300)